### PR TITLE
display gene source using gene metadata.name field from thoas

### PR DESCRIPTION
**Pull request by ens-ma7**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-958
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-957

## Importance
N/A

## Description
Displaying the gene source (symbol and name with URL) in Entity viewer and genome browser track drawer

## Deployment URL
http://xref-update.review.ensembl.org

## Views affected
Entity viewer and Genome Browser

### Other effects
N/A

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications
N/A

### CODE REVIEW

File: src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx [hunk 1]
```diff
@@ -63,6 +65,14 @@ const GENE_QUERY = gql`
           label
           value
         }
+        name {
+          accession_id
+          url
+          value
+          source {
+            name
+          }
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx:74]: Remove unused fields from the query.

File: src/ensembl/stories/shared-components/instant-download/InstantDownloadTranscript.stories.tsx [hunk 2]
```diff
@@ -34,6 +34,10 @@ const buildProteinCodingGene = () => {
         label: 'Protein coding',
         value: 'protein_coding',
         definition: 'Protein coding'
+      },
+      name: {
+        accession_id: 'HGNC:1097',
+        url: 'https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:1097'
```
Comment (azangru) [inline src/ensembl/stories/shared-components/instant-download/InstantDownloadTranscript.stories.tsx:40]: Why does InstantDownloadTranscript need to know about gene name metadata?

Comment (ens-ma7) [reply src/ensembl/stories/shared-components/instant-download/InstantDownloadTranscript.stories.tsx]: As discussed in code review, we dont need gene metadata at all, so removed the field completely. I have pushed the changes. Please review. Thanks

**Comment by jyothishnt** [comment]
wonder why the xref has jumped onto the next line where in XD it is alone with the Gene name?

**Comment by jyothishnt** [comment]
Note there is gap between the gene name and HGNC in the XD. Please could you add a margin?
Screenshot 2021-08-05 at 13 18 27

**Comment by azangru** [comment]
> wonder why the xref has jumped onto the next line where in XD it is alone with the Gene name?

Andrea specifically asked for it to be always placed on a new line ([link](https://genomes-ebi.slack.com/archives/C0MUF0C3Y/p1627917838023700?thread_ts=1627911010.014800&cid=C0MUF0C3Y))

**Comment by azangru** [comment]
Thoas is now stripping the `[Source: ...]` part from gene names, which means we don't need to. Thus, the formatter function:

https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/helpers/formatters/geneFormatter.ts#L17-L23

can be simplified to just handle the cases of no gene name, gene name that says "novel transcript", and all the rest of gene names.

**Comment by ens-ma7** [comment]
Thanks @jyothishnt and @azangru for the review. I have made the suggested changes, can you please review again. Thanks

**Review by jyothishnt — APPROVED** [review]

**Review by azangru — APPROVED** [review]

**Pull request merged**